### PR TITLE
[TACHYON-471] Rename Integration tests to include "IntegrationTest"

### PR DESCRIPTION
**Pull request by gpang**
Test names in 'integration-tests' should end with "IntegrationTest"

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/underfs/UnderFileSystemIntegrationTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /**
  * Unit tests for {@link UnderFileSystem}
  */
-public final class UnderFileSystemTest {
+public final class UnderFileSystemIntegrationTest {
```
Comment (calvinjia) [inline integration-tests/src/test/java/tachyon/underfs/UnderFileSystemIntegrationTest.java]: This one isn't really an integration test, is it?

**Comment by gpang** [comment]
I moved the UnderFileSystemTest to common, since it is not really an integration test.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**